src/test/java/divaeva/selenide/hw2/SearchBookByTittle.java

```diff
@@ -0,0 +1,55 @@
+package divaeva.selenide.hw2;
+
+import com.codeborne.selenide.CollectionCondition;
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.Selenide;
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class SearchBookByTittle {
+    public static void main(String[] args) {
+        openSite();
+        doLogin();
+        prepareSearchQuery();
+        ElementsCollection searchResult = submitSearchQuery();
+
+        Assert.assertEquals(searchResult.size(), 4);
```

Comment:
move this magic "4" to the separate variable and put it in the very beginning of the "main" method

Reply:
fixed